Dockerfile

```diff
@@ -2,9 +2,11 @@ FROM gcr.io/world-fishing-827/github.com/globalfishingwatch/gfw-pipeline:latest
 
 # Setup local application dependencies
 COPY . /opt/project
-RUN apt-get -qqy install gdal-bin libgdal-dev && \
+RUN  apt-get update && \
+  apt-get -qy install gdal-bin libgdal-dev && \
```

Comment:
@bitsofbits 
I think that a call about the changes in pipeline will be planned for this week.
Just as a summary: After reviewing and check that the custom template for Dataflow has a good impact in how we are running data in Dataflow and improved the times a lot, we decided to go with dataflow custom template. The big difference is that instead of let the workers are setup every time is invoked, downloads and installs each python package that the requirements says, we prepare an image with all the required packages needed, and don't need to spend time doing it.
We split the Dockerfile in 2, the Dockerfile-scheduler and the Dockerfile-worker. Dockerfile-scheduler has all dependencies needed to post the Dataflow Job. While the other has only the dependencies that the workers need.

Regarding the comment change, It's probably to move the libraries `gdal-bin` and `libgdal-deb`, that are not installed by pip to Dockerfile-scheduler. But would be better to have the call first.

Reply:
@smpiano , thanks for the info.  Regarding the custom dockerfile for Dataflow, I'll need to learn how to build my own container and launch that from the command line for development purposes.  Alternatively, I can run dataflow directly for development, but then I'd want to keep the launch times reasonable for that type of launch too (i.e., no pinned dataflow version).